src/config.ts

```diff
@@ -1,7 +1,7 @@
-import { cosmiconfig } from 'cosmiconfig';
 import JSON5 from 'json5';
+import { lilconfig } from 'lilconfig';
```

Comment:
what is the tradeoff for this. is lilconfig feature matched with cosmiconfig (given what we and users might depend on)?

Reply:
Feature match is the idea of `lilconfig` (but it has fewer dependencies and works faster). I moved from `cosmiconfig` to `lilconfig` in PostCSS Load Config and Size Limit without any problems.